src/cbc_syslog/core.py

```diff
@@ -266,3 +270,186 @@ def history(config, start, end, org_key=None):
                     break
 
     return success and output_success
+
+
+def wizard(output_file_path):
+    """
+    Setup Wizard Command
+
+    Args:
+        output_file_path (str): Output destination
+
+    Returns:
+        bool: Indicates short exit based on failure to execute
+    """
+
+    def valid_input(prompt, func, error_message=None):
+        """
+        Verify input and re-request if invalid.
+
+        Args:
+            prompt (str): The prompt to request input
+            func (lambda): The function to validate the response
+            error_message (str): The message to return in the case of failure
+        """
+        response = input(prompt)
+
+        if not func(response):
+            if error_message:
+                print(error_message + "\n")
+            else:
+                print("Invalid Input\n")
+            valid_input(prompt, func, error_message)
+        return response
+
+    with pathlib.Path(output_file_path).open("w+") as output_file:
+
+        output_file.write("[general]")
+        backup_dir = valid_input("Provide an absolute path to an existing backup directory: ",
+                                 lambda resp: pathlib.Path(resp).exists(),
+                                 "Directory not found")
+        output_file.write(f"backup_dir = \"{backup_dir}\"\n")
+
+        output_format = valid_input("What format would you like the data to be sent json or template: ",
+                                    lambda resp: resp.lower() in ["json", "template"]).lower()
+        output_file.write(f"output_format = \"{output_format}\"\n")
+
+        output_type = valid_input("How would you like to data to be sent (udp / tcp / tcp+tls / http / file): ",
```

Comment:
Maybe `the data`? Also maybe unify the messages on 313 and 317 - the options either in () or without

Reply:
Done